modules/virt-runbook-cdidataimportcronoutdated.adoc

```diff
@@ -4,11 +4,11 @@
 // * virt/logging_events_monitoring/virt-runbooks.adoc
 
 :_content-type: REFERENCE
-[id="virt-runbook-cdidataimportcronoutdated_{context}"]
+[id="virt-runbook-CDIDataImportCronOutdated"]
```

Comment:
I know this was discussed in the team chat about capitalizing, but I didn't remember seeing something about the "_{context}" part being removed. I know it only really matters if it's being reused, but most of our modules probably aren't reused, it's just good to follow the practice. FWIW, the CLI reference files that I auto-generate do include the context in the IDs (and the API docs probably should too).

Just want to make sure that @kalexand-rh is okay with this too before we proceed.

Reply:
@bergerhoffer @kalexand-rh 
The monitoring team plans to use the following format for the links in the Alerts:  `https://docs.openshift.com/container-platform/<version.X>.<version.Y>/virt/logging_events_monitoring/virt-runbooks.html#virt-runbook-<runbook-name>` 

If someone changes an assembly context (not likely but it could happen), that breaks all the links to runbooks in the UI and they cannot be fixed easily. We can control the module content by generating them with a script but we have no control over the assembly.

IMO, the potential risk of broken URLs hard-coded into our UI outweighs our best practices.

Reply:
@bergerhoffer 

@kalexand-rh agreed to allow the exception as long as I add a comment to each module that the module must not be reused.

Reply:
The context statement is a requirement to ensure that the modules can be reused. I think that Avital's argument is solid enough to grand an exception as long as we mark that the modules _cannot_ be reused. (We might need to revisit this issue when we move to toolchain.next. I am not sure if the URLs will be different with a different publication platform.)

Reply:
Cool thank you for confirming!